scripts/pick_one.coffee

```diff
@@ -0,0 +1,14 @@
+# Description:
+#   Example scripts for you to examine and try out.
+#
+# Command:
+#   Hobbes: pick one: Sitar, Zoi's, Tikkaway
+#   Dan: Zoi's
+
+module.exports = (robot) ->
+
+  robot.respond /pick one: (.*)$/i, (msg) ->
+    choiceText = msg.match[1]
+    choices = choiceText.split(",")
+    choice = msg.random choices
+    msg.reply choice
```

Comment:
we can probably just do `msg.reply msg.random choices` here, but I may be wrong about that. My :coffee: script is :poop: